commit 5e48deb25e8c1f2ee596120ed5bcbdb6daf3dee7 (refs/changes/21/4421/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-25T17:28:16+00:00 (1 year, 3 months ago)
    
    MANTA-3787 waferlock should look for "zookeeper" as well as "binder"
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/sapi_manifests/waferlock/template b/sapi_manifests/waferlock/template
index 3b61b52..7609bf9 100644
--- a/sapi_manifests/waferlock/template
+++ b/sapi_manifests/waferlock/template
@@ -26,7 +26,9 @@
 			"{{{moray_domain}}}",
 			"{{{BINDER_SERVICE}}}",
 			{{! Don't use pgstatsmon_domain, might not be set. }}
-			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}"
+			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}",
+			{{! Same deal here: this is for legacy DCs. }}
+			"zookeeper.{{{datacenter_name}}}.{{{dns_domain}}}"
 		],
 		"sapi_url": "{{{sapi-url}}}",
 		"sapi_services": ["sdc/moray"],
